### PR TITLE
Fix icon attrs default values

### PR DESCRIPTION
**Pull request by martinRenou**
Workaround for the discussion in #634

### CODE REVIEW

File: ipyleaflet/leaflet.py [hunk 1]
```diff
@@ -260,11 +260,24 @@ class Icon(UILayer):
...
-    icon_size = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    shadow_size = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    icon_anchor = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    shadow_anchor = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    popup_anchor = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
+    icon_size = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    shadow_size = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    icon_anchor = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    shadow_anchor = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    popup_anchor = List([0, 0], allow_none=True).tag(sync=True, o=True)
```
Comment (davidbrochart) [inline ipyleaflet/leaflet.py:267]: Why is the default value not `None`?

File: ipyleaflet/leaflet.py [hunk 2]
```diff
@@ -260,11 +260,24 @@ class Icon(UILayer):
...
+
+    @validate('icon_size', 'shadow_size', 'icon_anchor', 'shadow_anchor', 'popup_anchor')
+    def _validate_attr(self, proposal):
+        value = proposal['value']
+
+        # Workaround Traitlets which does not respect the None default value
+        if len(value) == 0:
```
Comment (davidbrochart) [inline ipyleaflet/leaflet.py:274]: Won't it fail if the value is `None`?

File: ipyleaflet/leaflet.py (hunk 1 above)
Comment (martinRenou) [reply ipyleaflet/leaflet.py:267]: I am trying to respect the LeafletJS default values: https://leafletjs.com/reference-1.6.0.html#icon-popupanchor

File: ipyleaflet/leaflet.py (hunk 2 above)
Comment (martinRenou) [reply ipyleaflet/leaflet.py:274]: No, Traitlets seems to have set the value to an empty list `[]` in the meantime.

File: ipyleaflet/leaflet.py (hunk 1 above)
Comment (davidbrochart) [reply ipyleaflet/leaflet.py:267]: :+1:

File: ipyleaflet/leaflet.py (hunk 2 above)
Comment (davidbrochart) [reply ipyleaflet/leaflet.py:274]: :+1:

Comment (martinRenou) [reply ipyleaflet/leaflet.py:274]: Maybe this line should be `if value is None or len(value) == 0:` to handle the case when Traitlets will be fixed, if that happens

Comment (davidbrochart) [reply ipyleaflet/leaflet.py:274]: Yes, that would be safer.

File: ipyleaflet/leaflet.py (hunk 1 above)
Comment (martinRenou) [reply ipyleaflet/leaflet.py:267]: This makes me think I forgot to update them JS side #637

**Pull request merged**